flow-typed/npm/danger_v0.x.x.js

```diff
@@ -0,0 +1,193 @@
+
```

Comment:
It would be really awesome if this reached [flowtype/flow-typed](https://github.com/flowtype/flow-typed) :v: